pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -50,6 +50,11 @@ type OperatorSourceSpec struct {
 type OperatorSourceStatus struct {
 	// Current phase of the OperatorSource object
 	CurrentPhase ObjectPhase `json:"currentPhase,omitempty"`
+
+	// Packages is a comma separated list of package(s) each of which has been
+	// downloaded and processed by Marketplace operator from the specified
+	// endpoint.
+	Packages string `json:"packages,omitempty"`
```

Comment:
Is this required to keep the datastore cache in sync? If not, this warrants its own PR as it is introducing a new status field. If it does then it needs to be explained in the commit message as to how it is connected. 

My suggestion is that this becomes a separate PR so that it is clearly called out. You can cherry pick this into another branch and drop it from this PR.